src/charm.py

```diff
@@ -5,6 +5,8 @@
 
 """Charm Jenkins."""
 
+# pylint: disable=too-many-instance-attributes
```

Comment:
Should this include a note as to why it is reasonable to disable this?

Reply:
Removed. It's not needed anymore